extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/ElasticsearchCommonBuildTimeConfig.java

```diff
@@ -133,9 +133,61 @@ interface ElasticsearchDevServicesBuildTimeConfig {
         @WithDefault("true")
         boolean reuse();
 
+        /**
+         * Kibana configuration for Dev Services.
+         */
+        @ConfigDocSection
+        DashboardDevServicesConfig dashboard();
+
         enum Distribution {
             ELASTIC,
             OPENSEARCH
         }
+
+        @ConfigGroup
+        interface DashboardDevServicesConfig {
+            /**
+             * Whether the Dashboard Dev Service should start with the application in dev mode or tests.
+             * <p>
+             * Kibana Dev Services are disabled by default when Elasticsearch Dev Services are enabled.
+             *
+             * @asciidoclet
+             */
+            @WithDefault("false")
+            boolean enabled();
+
+            /**
+             * Optional fixed port the dev service will listen to.
+             * <p>
+             * If not defined, the port will be chosen randomly.
+             */
+            Optional<Integer> port();
+
+            /**
+             * The Elasticsearch container image to use.
+             *
+             * Defaults depend on the configured `distribution`:
+             *
+             * * For the `elastic` distribution: {kibana-image}
+             * * For the `opensearch` distribution: {dashboardimage}
+             *
+             * @asciidoclet
+             */
+            Optional<String> imageName();
+
+            /**
+             * Environment variables that are passed to the container.
+             */
+            @ConfigDocMapKey("environment-variable-name")
+            Map<String, String> containerEnv();
+
+            /**
+             * The value for the NODE_OPTIONS env variable.
+             *
+             * @asciidoclet
+             */
+            @WithDefault(" ")
```

Comment:
Small question: why do we pass an empty value here? Shouldn't it be an `Optional` instead?